packages/solid-crs-core/tsconfig.json

```diff
@@ -4,8 +4,9 @@
     "baseUrl": "lib",
     "outDir": "dist",
     "emitDeclarationOnly": true,
+    "resolveJsonModule": true,
     // Recommended by Lit Element
-    "module": "es2015",
+    "module": "es2020",
     "target": "es2017",
```

Comment:
is `es2020` needed? If only 2018 is needed i would go for that.
Also remove teh comment as it does not apply anymore.

Reply:
Probably does not matter, but the `presentation` and `core` packages use it as well.